frontend/src/components/CustomButton/CustomButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { forwardRef, Ref } from "react";
+
+import { CustomButtonProps } from "@/components/CustomButton/customButton.types";
+
+export const CustomButton = forwardRef(
+  (
+    { className, children, ...props }: CustomButtonProps,
+    ref: Ref<HTMLButtonElement>
+  ) => {
+    return (
+      <button
+        type="button"
+        className={`bg-black text-white px-8 py-4 mt-3 font-bold rounded-lg ${className}`}
+        ref={ref}
+        {...props}
+      >
+        {children}
+      </button>
+    );
+  }
+);
```

Comment:
The best way to handle a custom button is to add a small debounce on click.

[More Information](https://sandny.com/2021/02/16/usidebounce-with-react-hooks/)